tests/run-fvttests.sh

```diff
@@ -95,7 +95,34 @@ kubectl -n ${CLUSTER_NAMESPACE} cp "${POD_NAME}:/bzt-configs/tests/results" "${O
 echo "Copying test results log files and screenshots to ${OUTPUT_DIR}/artifacts"
 kubectl -n ${CLUSTER_NAMESPACE} cp "${POD_NAME}:/tmp/artifacts" "${OUTPUT_DIR}/artifacts"
 
-# First Check to see if python3 is available for use with xmlCombiner.py script. If not first install python3.8 using yum.
+######################################### START of section specific to SPARK based fvt tests ######################################################
+
+#Give a name to the new pod that will be spun off to be used to execute spark-submit jobs for HI FVT testing.
+SPARK_POD_NAME=engine-spark-fvt-test
+
+# Call a script "create-sparkfvt-pod.sh" to prepare and then spin off the new pod to execute spark-submit job commands
+bash tests/create-sparkfvt-pod.sh ${SPARK_POD_NAME}
+
+# Clean up any Spark Driver pod whose name starts with "cohortfvt-spark" that is hanging around from a previous spark-submit run before doing a new spark-submit.
+DVRNAMEPREFIX="cohortfvt-spark"
+olddriverpod=`kubectl -n ${CLUSTER_NAMESPACE} get pods --no-headers -o custom-columns=":metadata.name" | grep ${DVRNAMEPREFIX} | grep driver`
+kubectl -n ${CLUSTER_NAMESPACE} delete pod ${olddriverpod}
+
+# Build up the spark-submit command arguments as a string
+sparkargs="--deploy-mode cluster --name cohortfvt-spark --class com.ibm.cohort.cql.spark.SparkCqlEvaluator  --conf spark.sql.shuffle.partitions=10 --conf spark.default.parallelism=10 local:///opt/spark/jars/cohort-evaluator-spark-1.0.2-SNAPSHOT.jar -d /cohort-config/fvt/fvt-context-definitions.json -j /cohort-config/fvt/fvt-cql-jobs.json -m /cohort-config/fvt/fvt-model-info.xml -c /cohort-config/fvt/cql --input-format delta -i Device=s3a://cohort-data-tenant2/fvt-input-data/Device -i Observation=s3a://cohort-data-tenant2/fvt-input-data/Observation -i Patient=s3a://cohort-data-tenant2/fvt-input-data/Patient -i PatientDeviceJoin=s3a://cohort-data-tenant2/fvt-input-data/PatientDeviceJoin -i Practitioner=s3a://cohort-data-tenant2/fvt-input-data/Practitioner --overwrite-output-for-contexts -o Patient=s3a://cohort-data-tenant2/fvt-output/Patient_cohort -o Observation=s3a://cohort-data-tenant2/fvt-output/Observation_cohort -o Practitioner=s3a://cohort-data-tenant2/fvt-output/Practitioner_cohort -o Device=s3a://cohort-data-tenant2/fvt-output/Device_cohort"
```

Comment:
`spark.sql.shuffle.partitions` and `spark.default.parallelism` probably belong in spark-defaults.conf unless you plan on changing them between test runs. It isn't required, but you might consider storing your program arguments in a file that you copy into the running POD. It would greatly simplify this string.